instaclustr/resource_cluster.go

```diff
@@ -1308,6 +1348,7 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error {
 	d.SetId(cluster.ID)
 	d.Set("cluster_id", cluster.ID)
 	d.Set("cluster_name", cluster.ClusterName)
+	d.Set("cdc_id", cluster.CdcId)
```

Comment:
Not sure if "CDC" is really a public friendly term. Why not `data_centre_id` ? 
Same goes for `target_*_cdc_id`s

Should also document the read only data centre ID field under docs/resources/cluster.md

Reply:
Since there's a mix of multi-DC and single DC schemas in this resource, this should probably even be `default_data_centre_id` to make it clear.

Reply:
Hey Arjun,

Re: "Should also document the read only data centre ID field under docs/resources/cluster.md"

Are you thinking that this should be in the "Schema" section on that page? Those appear to be input fields, and not read only documented there though, as there are other fields that are read only/computed and not documented there though, such as 'cluster_id'.

Happy to add it either way, just wanted to check.

Thanks,
Russ

Reply:
Yeah we should document read-only properties too. You could just mention that they are Read-only in the description and default columns. Other read-only inputs definitely should have also been documented :)